src/js/helpers.test.js

```diff
@@ -185,61 +154,43 @@ echo "Done!"'
         ...window.location,
         hostname: 'staging.hosted.mender.io'
       };
+      jest.clearAllMocks();
+      const cookies = new Cookies();
+      cookies.get.mockReturnValue('omnomnom');
+      cookies.set.mockReturnValueOnce();
     });
     afterEach(postTestCleanUp);
 
     it('should contain sane information for staging calls', async () => {
       code = getDebConfigurationCode(undefined, true, false, 'token', 'raspberrypi3');
       expect(code).toMatch(
-        `wget -q -O- https://get.mender.io | sudo bash -s -- --demo --commercial --jwt-token "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJjZTNkMGY4Yy1hZWRlLTQwMzAtYjM5MS03ZDUwMjBlYjg3M2UiLCJzdWIiOiJhMzBhNzgwYi1iODQzLTUzNDQtODBlMy0wZmQ5NWE0ZjZmYzMiLCJleHAiOjE2MDY4MTUzNjksImlhdCI6MTYwNjIxMDU2OSwibWVuZGVyLnRlbmFudCI6IjVmODVjMTdiY2U2MmI3ZmE3ZjVmNzA0MCIsIm1lbmRlci51c2VyIjp0cnVlLCJpc3MiOiJNZW5kZXIgVXNlcnMiLCJzY3AiOiJtZW5kZXIuKiIsIm1lbmRlci5wbGFuIjoicHJvZmVzc2lvbmFsIiwibmJmIjoxNjA2MjEwNTY5fQ.qVgYdCzLTf8OdK9uUctqqaY_HWkIiwpekuGvuGQAXCEgOv4bRNDlZRN_ZRSbxQoARG3pquhScbQrjBV9tcF4irTUPlTn3yrsXNO17DpcbTVeKRkb88RDtIKiRw3orVZ_GlIb-ckTQ5dS-Nqlyyf3Fmrhca-gwt6m_xv2UrmJK6eYYTMfggdRRWb-4u7mEkBI_pHPMTQrT8kJ2BeX-vHgazH9AoH0k85LHtFZQXD7pXHlDZRnLxJXukncwMGDmF17374gavYAIyDIzcC8sEBMDnVXgpikeA1sauzirqix6mAVs6XmxdQO7aF0wfXO1_PTYUA3Nk1oQfMYNlEI3U9uLRJRZIq2L8fmrrBryhstKd4y0KlBbGAQrx8NtRkgajjd1ljMfPBUEZrb7uSerVjneiO-aIBO76CuH0zdklphIjpGJeogkBhe8pAYNggp1XsZHgpZfl7IE5faKaDkMGnutaea--Czor6bhqUNCuY4tR0cpQJbNwy6LS9o1CFy4Log" && \\
-sudo bash -c 'DEVICE_TYPE="raspberrypi3" && \\
-TENANT_TOKEN="token" && \\
-echo "Running mender setup for staging.hosted.mender.io" && \\
-mender setup \\
-  --device-type $DEVICE_TYPE \\
-  --quiet --hosted-mender \\
-  --tenant-token $TENANT_TOKEN \\
-  --retry-poll 30 \\
-  --update-poll 5 \\
-  --inventory-poll 5 && \\
-systemctl restart mender-client && \\
-(cat > /etc/mender/mender-connect.conf << EOF
-{
-  "ServerCertificate": "/usr/share/doc/mender-client/examples/demo.crt",
-  "User": "pi",
-  "ShellCommand": "/bin/bash"
-}
-EOF
-) && systemctl restart mender-connect && \\
-echo "Done!"'
-`
+        `wget -q -O- https://get.mender.io | sudo bash -s -- --demo --commercial --jwt-token "omnomnom" -- --quiet --device-type "raspberrypi3" --demo --hosted-mender --tenant-token "token"`
```

Comment:
This test now is exactly the same as the one above (before the only difference was the user friendly message "Running mender setup for staging.hosted.mender.io") so I'll remove it.